test/API/H5_api_link_test.c

```diff
@@ -25887,6 +25464,13 @@ link_iter_external_links_cb(hid_t group_id, const char *name, const H5L_info2_t
     char    expected_link_name[LINK_ITER_EXT_LINKS_TEST_BUF_SIZE];
     herr_t  ret_val = H5_ITER_CONT;
 
+    if (!(vol_cap_flags_g & H5VL_CAP_FLAG_EXTERNAL_LINKS)) {
+        SKIPPED();
+        HDprintf("    API functions for external links aren't supported with this "
+                 "connector\n");
+        return 0;
```

Comment:
All the callback functions in this file should probably return 1 if they don't support the capability flag being checked for

Reply:
Okay.